### ISSUE TITLE
Jitters 

### ISSUE THREAD

**Issue by Redline676767** [Quaver/Quaver#3148]
Okay the game just jitters I'm running it under linux and it works fine for the most part but when I open a song theres sometimes jitters and I miss notes because its like it skips them 

https://www.youtube.com/watch?v=kpZQD2lNXo0 just like these videos

**Comment by Malachi-Buchheit** [comment]
Hey, I'm the one who made that video.

It could be an issue with how notes are being loaded, as my 4GB laptop had this issue when I tried to play on it. I toyed around with it on my laptop, and I could reproduce the issue with ~30% CPU and ~2/3.9GB RAM usage. Having high CPU usage (90%) seemed to make it worse, while having high RAM usage (~3.2/3.9) just froze the computer without causing jitter.

I'm using MX linux on both the computer I used to record that video and also my laptop.
